main.go

```diff
@@ -42,6 +42,7 @@ type tcpEventV4 struct {
 	Type  uint32
 	Pid   uint32
 	Comm  [16]byte
+	Ip    uint8
```

Comment:
could this be named IpVersion?

Also, is the struct packed? I wonder if the uin8 would break the alignment.